wallet/wallet-rpc-lib/src/rpc/interface.rs

```diff
@@ -774,6 +774,16 @@ trait WalletRpc {
         options: TransactionOptions,
     ) -> rpc::RpcResult<NewTransaction>;
 
+    /// Freeze an order given its id. This prevent an order from being filled.
+    /// Only conclude operation is allowed afterwards.
```

Comment:
prevent an order -> prevents the order
conclude operation -> a conclude operation